experimental/dds/tree2/src/shared-tree/sharedTreeView.ts

```diff
@@ -55,6 +56,11 @@ export interface ViewEvents {
 	 * This is mainly useful for knowing when to do followup work scheduled during events from Anchors.
 	 */
 	afterBatch(): void;
+
+	/**
+	 * An undoable change has been made to the tree. This is used to track undo/redo on the {@link SharedTreeViewUndoRedoHandler}
+	 */
+	undoable(type: UndoRedoManagerCommitType, target: ISharedTreeView): void;
```

Comment:
I actually don't care one way or another. I chose to do it this way only because this is how the other undo redo handlers do it. I think this is so that the same handler can be used for multiple trees so it's probably cleaner on the client side.